hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheService.java

```diff
@@ -288,6 +289,7 @@ public void deleteCache(String name, String callerUuid, boolean destroy) {
         CacheConfig config = deleteCacheConfig(name);
         if (destroy) {
             destroySegments(name);
+            sendInvalidationEvent(name, null, SOURCE_NOT_AVAILABLE);
```

Comment:
Do we need to send this event? As I trace this call, it can be called either via reset method (but this method already has invalidation at the end), or ProxyServiceImpl.destroyLocalDistributedObject. Do we need invalidation for ProxyServiceImpl.destroyLocalDistributedObject?